documentation/3.4.0/setup/docker/stand-alone/ubuntu/README.md

```diff
@@ -0,0 +1,226 @@
+<div align="center">
+
+# Projects Service
+
+<a href="https://shikshalokam.org/elevate/">
+<img
+    src="https://shikshalokam.org/wp-content/uploads/2021/06/elevate-logo.png"
+    height="140"
+    width="300"
+  />
+</a>
+
+![GitHub package.json version (subfolder of monorepo)](https://img.shields.io/github/package-json/v/ELEVATE-Project/mentoring?filename=src%2Fpackage.json)
+[![License](https://img.shields.io/badge/license-MIT-blue.svg)](https://opensource.org/licenses/MIT)
+
+</details>
+</details>
+
+</br>
+The Project building block facilitates the creation and engagement with micro-improvement projects.
+
+</div>
+</br>
+
+# Supported Operating Systems
+
+-   **Ubuntu (Recommended: Version 20 and above)**
+
+# Setup Options
```

Comment:
@MallanagoudaB there only one setup option described in this file. then why we need this?